public/index.ejs

```diff
@@ -267,10 +291,22 @@ function updateFollowingUI(users) {
       e.preventDefault();
       history.pushState({path: newURL.toString()}, '', newURL.toString());
       updateUIToMatchURL();
+      fetchAbortHandler();
```

Comment:
Is this needed since we're already calling it from `updateUIToMatchURL()`?

Reply:
I believe this is outdated in the latest commit, yeah is being called already.

Reply:
It doesn't look like this is resolved.

Reply:
Ah, missed it, my mistake.  Apologies.